apps/fluent-tester/src/TestComponents/Divider/DividerDefault.tsx

```diff
@@ -0,0 +1,45 @@
+import { Divider } from '@fluentui-react-native/divider';
+import { TextV1 as Text } from '@fluentui-react-native/text';
+import * as React from 'react';
+import { View } from 'react-native';
+
+const DividerWithInset = Divider.customize({ insetSize: 56, thickness: 3 });
+
+export const DividerDefault: React.FunctionComponent = () => {
+  return (
+    <View style={{ paddingVertical: 8 }}>
```

Comment:
Please define objects as consts (or styles as const stylesheet objects) and avoid passing in object literals as props.

See here for more info: https://www.debugbear.com/blog/react-rerenders#passing-objects-as-props